shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Pool.hs

```diff
@@ -104,6 +105,10 @@ instance
   type BaseM (POOL era) = ShelleyBase
   type PredicateFailure (POOL era) = PoolPredicateFailure era
 
+  data Event (POOL era)
+    = NewPoolParam
+    | NewFuturePoolParam
```

Comment:
Is the plan to eventually add more information to these? I think just the pool ID is all we really need.